pkg/network/http/monitor.go

```diff
@@ -33,6 +33,7 @@ type Monitor struct {
 	batchManager           *batchManager
 	batchCompletionHandler *ddebpf.PerfHandler
 	telemetry              *telemetry
+	lastTelemetry          chan telemetry
```

Comment:
WDYT about naming this `lastTelemetryDelta`? or `telemetrySnapshot` ?

Reply:
I have no preference tbh. Let's go for `telemetrySnapshot`.